packages/lesswrong/lib/sql/Query.ts

```diff
@@ -308,6 +309,15 @@ abstract class Query<T extends DbObject> {
       if (op === "=" && this.isCaseInsensitive && typeof value === "string") {
         return [`LOWER(${resolvedField}) ${op} LOWER(`, new Arg(value), ")"];
       }
+      /**
+       * `<>` returns null if either of the compared values are null
+       * This is generally not the result you want when checking whether a field is $ne to a specific value
+       * So for nullable fields (most of them, so far) use `IS DISTINCT FROM`
+       * This will return records where e.g. the field is null and you want everything not equal to 5.
+       */
+      if (!(fieldType instanceof NotNullType) && op === "<>") {
```

Comment:
As I mentioned in slack, I'm not sure why I didn't run into this issue the first time I was trying to migrate RSSFeeds (possibly I did and then forgot that I needed to patch it?), but the query in the rss-integration cron job (`RSSFeeds.find({status: {$ne: 'inactive'}})`) was returning 0 feeds to update... because `$ne` got translated to `<>` (unless the value it was being compared with was itself `null` or a boolean, in which case it uses `IS` or `IS NOT`), which returns false if the value of the column is `null`.

I'd expect this to be causing issues in other cases where we have nullable columns that aren't boolean types and we're checking that the value isn't equal to something other than `null`, but those could be broken in more subtle ways.